### PR TITLE
Add arachni to vulndb convertor

### PR DESCRIPTION
**Pull request by m0sth8**

### CODE REVIEW

File: convertors/arachni.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+DEFAULT_EFFORT = 50
+
+
+def slugify(filename):
+    filename = re.sub(r'\W+', '-', filename.lower())
+    return filename.strip("-")
+
+
+def convert_multi_string(text):
```
Comment (andresriancho) [inline convertors/arachni.py:21]: Seems like this is the function that's breaking the \n and spaces in https://github.com/vulndb/data/pull/9

**Comment by andresriancho** [comment]
Don't send PRs for this stuff, simply commit and push, it's your repo too :+1:

File: convertors/arachni.py (hunk 1 above)
Comment (m0sth8) [inline convertors/arachni.py:21]: Yes, It was there. Default behavior is very strict, but all these options "expand_tabs=False, break_on_hyphens=False, replace_whitespace=False, break_long_words=False"  help me =)

**Comment by m0sth8** [comment]
I haven't used python for a long time. So, some code review is good for me

File: convertors/arachni.py (hunk 1 above)
Comment (andresriancho) [inline convertors/arachni.py:21]: > Yes, It was there.

It was means you already fixed it?

Comment (m0sth8) [inline convertors/arachni.py:21]: Yep, you can see it in latest data commit https://github.com/vulndb/data/pull/9

**Pull request merged**